concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -106,8 +108,7 @@ class ValidateGolemMessageSignedWithKeyUnitTest(TestCase):
     def test_validate_golem_message_signed_with_key_should_not_raise_error_if_correct_message_and_key_is_used(self):
         ping = message.Ping()
 
-        dumped_ping = dump(ping, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
-        ping = load(dumped_ping, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+        ping.sign_message(CONCENT_PRIVATE_KEY)
```

Comment:
We want to test if dumping and loading create correct signature, so please keep dumping and loading but use different message.
Please assert that this message class attribute `SIGN` is `False`.
In `validate_golem_message_signed_with_key` please use `message.verify` instead of `ecdsa_verify()`.

Reply:
You mean `message.verify_signature(public_key)` ?

Reply:
Yes, my mistake.

Reply:
Okey, done.